libs/agno/agno/workflow/v2/loop.py

```diff
@@ -131,7 +131,9 @@ def execute(
         user_id: Optional[str] = None,
     ) -> List[StepOutput]:
         """Execute loop steps with iteration control - mirrors workflow execution logic"""
-        logger.info(f"Executing loop: {self.name} (max_iterations: {self.max_iterations})")
+        # Use workflow logger for loop orchestration
+        use_workflow_logger()
```

Comment:
I dont think we need to use `use_workflow_logger()` everywhere...
like in team also we have this private function 
```py
def _set_debug(self) -> None:
        if self.debug_mode or getenv("AGNO_DEBUG", "false").lower() == "true":
            self.debug_mode = True
            set_log_level_to_debug(source_type="team", level=self.debug_level)
        else:
            set_log_level_to_info(source_type="team")
```

which we call just once during initialization which automatically takes care of log_debug for teams, i think we should do something similar for workflows as well